.github/workflows/github_release.yml

```diff
@@ -59,16 +58,44 @@ jobs:
           reno report --no-show-source --ignore-cache --version v${{ steps.version.outputs.current_release }} -o relnotes.rst
 
       - name: Convert to Markdown
-        uses: docker://pandoc/core:3.1
+        uses: docker://pandoc/core:3.8
         with:
-          args: "--from rst --to markdown_github --no-highlight relnotes.rst -o relnotes.md --wrap=none"
+          args: "--from rst --to gfm --syntax-highlighting=none --wrap=none relnotes.rst -o relnotes.md"
```

Comment:
Here we can also do something like:

```
"--from rst --to gfm --syntax-highlighting=none --wrap=none relnotes.rst -o enhanced_relnotes.md"
``` 

to save the copy step below. Your choice!

Reply:
Ah, unfortunately pandoc creates a markdown files that cannot be later modified (wrong permissions).
I also tried to run `chmod` and I got other permission issues. I am creating the copy for this reason.